src/main/java/org/blondin/fa/Main.java

```diff
@@ -33,6 +33,8 @@ private Main() {
      */
     public static int main(String[] args) throws IOException {
 
+        System.out.println("This is a SonarQube issue with https://sonarqube.com/");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)